### ISSUE TITLE
Saplings via tree_populator don't work

### ISSUE THREAD

**Issue by evilsaloon** [PluginBugs/Issues-ItemsAdder#2773]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

evilsaloon#3438

### What happened?

Right clicking a sapling doesn't spawn a tree at all.

Could be an extension of #2758 and #2759.

### Steps to reproduce the issue

1. Add item .yml
2. /iareload
3. Try to right click sapling to place a tree
4. It doesn't place

### Server version

Current: git-Purpur-1996 (MC: 1.20.1)*
Previous: git-Purpur-1995 (MC: 1.20.1)
* You are running the latest version

### ItemsAdder Version

ItemsAdder version 3.5.0-r2

### ProtocolLib Version

ProtocolLib version 5.1.0-SNAPSHOT-654

### LoneLibs Version

LoneLibs version 1.0.25

### FULL server log

https://pastebin.com/XVjFACtR

### Error (optional)

```shell
No error in console, but when trying to place a tree via /iaspawntree, I get the following in chat:

[16:05:51] [Render thread/INFO]: [CHAT] [ItemsAdder] Now rightclick the surface on where you want to spawn the tree
[16:05:52] [Render thread/INFO]: [CHAT] [ItemsAdder] Can't spawn a tree here!
```

### Problematic items yml configuration file (optional)

```shell
info:
  namespace: testserv
items:
  custom_tree_sapling:
    nbt: "{MYTHIC_TYPE:\"custom_tree_sapling\", display:{Name:'{\"italic\":false,\"translate\":\"item.testserv.custom_sapling\"}'}}"
    permission: testserv
    resource:
      generate: true
      material: PAPER
      textures:
      - block/custom_sapling.png
    behaviours:
      sapling:
        tree_populator: custom_tree
trees_populators:
  custom_tree:
    worlds:
    - world
    bottom_blocks:
    - DIRT
    - GRASS_BLOCK
    - PODZOL
    - SNOW
    - SNOW_BLOCK
    chance: 40.0
    max_height: 100
    min_height: 50
    amount: 1
    iterations: 1
    tree_type: TREE
    leaves: testserv:custom_leaves
    log: testserv:custom_log
```

**Comment by yorik100** [comment]
I confirm, it's broken

**Comment by yorik100** [comment]
Fixed in new version!

**Comment by evilsaloon** [comment]
+1, fixed in 3.5.0b
